src/config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * General configuration
+ *
+ * @class config
+ * @memberof dc
+ * @returns {dc.config}
+ */
+dc.config = (function () {
+    var _config = {};
+
+    var _useV2CompatColorScheme = false;
+    _config.compatColors = null;
+    var _schemeCategory20c = [
+        '#3182bd', '#6baed6', '#9ecae1', '#c6dbef', '#e6550d',
+        '#fd8d3c', '#fdae6b', '#fdd0a2', '#31a354', '#74c476',
+        '#a1d99b', '#c7e9c0', '#756bb1', '#9e9ac8', '#bcbddc',
+        '#dadaeb', '#636363', '#969696', '#bdbdbd', '#d9d9d9'];
+
+    /**
+     * Set this flag to use DCv2's color scheme (d3.schemeCategory20c).
+     * Please note this color scheme has been deprecated by D3
+     * (https://github.com/d3/d3/blob/master/CHANGES.md#changes-in-d3-50) as well as DC.
+     * @method useV2CompatColorScheme
```

Comment:
Let's add an example of what you should do if you want to keep the old colors and not get the warning. Namely, you'd copy `_schemeCategory20c` and apply it to each chart.

Alternately, since we are adding a level of indirection here, we could rename `compatColors` to `defaultColors` and make it settable and permanent, so that users could automatically have the same color set for all charts. That might be a good idea if it turns out that `schemeSet1` was the wrong choice.

Reply:
If we allow setting defaultColors, we need to add in the API docs. In addition we may create an example to show how to use their color scheme.